targets/nuttx-stm32f4/Makefile

```diff
@@ -52,10 +52,12 @@ updateobjs:
 
 .PHONY: cleanlibs
 cleanlibs: updateobjs
-	rm -f $(OBJS)
+	rm -f $(OBJS) $(LIBS)
 
 clean: cleanlibs
 
+all: cleanlibs
```

Comment:
This line should be after the `include $(APPDIR)/Application.mk` line. First, the `Application.mk::all` rule should be executed, and the cleanup should only happen after that.